tests/test_processing.py

```diff
@@ -1,12 +1,23 @@
+import numpy as np
+import pytest
+
 from glidertools.processing import (
     calc_backscatter,
     calc_fluorescence,
     calc_oxygen,
     calc_par,
     calc_physics,
 )
+from tests.test_physics import dat
 
 
-def test_dummy():
-    """WE REALLY NEED TO ADD TESTS!!! THESE JUST TEST THE BASIC IMPORT!!!"""
-    assert 1 == 1
+def test_calc_oxygen():
+    o2ml, o2pc, o2au = calc_oxygen(
+        dat.oxy_raw, dat.pressure, dat.salt_raw, dat.temp_raw
+    )
+    assert np.nanmin(o2ml) == pytest.approx(3.7152995, 0.0001)
```

Comment:
I usually use numpys `np.testing.assert_allclose`. I am curious if this is doing something different? I guess the error bound id larger than machine precision.

Reply:
I wouldn't expect they differ much. Happy to switch to the numpy version if that's to be the repo standard

Reply:
I think this should be fine for single values like these. The numpy version works well for n-dimensional arrays, but might be overkill here. No need to change, just curious.

Reply:
@callumrollo can you explain what the difference between o2sat and o2pc is? I got confused there.

Reply:
Hi @soerenthomsen I'm no expert in oxygen, but the two terms are different ways to relate observed oxygen to the oxygen saturation of the water.  Oxygen saturation is a percentage (it can be greater than 100 with bubbles though I believe) and aou is apparent oxygen utilisation, I believe it will be positive or negative for net production/consumption by ocean critters. They are calculated at line 251 as:
```
o2aou = o2sat - o2mll
o2pct = o2mll / o2sat * 100
```
where o2mll is observed oxygen and o2sat is oxygen saturation